linkerd/docs/config.md

```diff
@@ -37,6 +37,17 @@ namers:
 tracers:
 - kind: io.l5d.zipkin
   sampleRate: 0.02
+
+telemetry:
+- kind: io.l5d.commonMetrics
+- kind: io.l5d.statsd
```

Comment:
should we mark some of the kind" "experimental" 
[linkerd doc example](https://linkerd.io/config/0.8.4/linkerd/index.html#informational-response-headers)

> experimental	false	Set this to true to opt-in to experimental XXX support.